.meteor/versions

```diff
@@ -11,6 +11,7 @@ babel-runtime@1.5.0
 base64@1.0.12
 binary-heap@1.0.11
 boilerplate-generator@1.7.1
+bundle-visualizer@1.2.3
```

Comment:
Probably don't actually want to check this in.

Reply:
Gah, damnit. Fixed